Automattic/simplenote-ios
# TransitionController: Nukes Magic Numbers

### Details: 
This PR removes (LOTS) of magic numbers embeded in `SPTransitionController`. **No new functionality is being added here.**

@SergioEstevao may I bug you with this one sir?
Thanks in advance!!

Ref. #486

### Test
1. Launch Simplenote
2. Press over any rows in the Notes List
3. Verify the animation looks great
4. Press Back
5. Verify the animation still looks great!

### Release
These changes do not require release notes.
